relativedeltafield/__init__.py

```diff
@@ -75,6 +97,45 @@ def format_relativedelta(relativedelta):
 		return 'P{}'.format(result_big)
 
 
+class RelativeDeltaFormField(forms.CharField):
```

Comment:
Could you add tests for the form field? This package is relatively well-tested, and I don't use Django admin myself, so without tests I'm definitely afraid to break things if I change anything.

Reply:
Will do